resources/zone_id.js

```diff
@@ -271,7 +271,7 @@ const ZoneId = {
   'TheCrystarium': 819,
   'TheDancingPlague': 845,
   'TheDancingPlagueExtreme': 858,
-  'TheDiadem': 929,
+  'TheDiadem': [901, 929], // v5.21, v5.31
```

Comment:
Unfortunately, as the top of this file tries to mention:
```
// Auto-generated from gen_zone_id_and_info.py
// DO NOT EDIT THIS FILE DIRECTLY
```

I think you need to update the script to generate this, otherwise the next time anybody runs `util/gen_zone_id_and_info.py` it will remove this change.

I also think for consistency, maybe the script could emit something like `'TheDiadem521': 901, 'TheDiadem': 929`, and then in the trigger file you could say `zoneId: [ ZoneId.TheDiadem521, ZoneId.TheDiadem],`.  That way all of the ZoneIds are all just a number, consistently.

Regardless, I think the script itself needs to be updated to emit these values, sorry.